inference-engine/src/mkldnn_plugin/nodes/mkldnn_deconv_node.cpp

```diff
@@ -240,6 +245,11 @@ void MKLDNNDeconvolutionNode::setPostOps(mkldnn::primitive_attr &attr) {
             eltwiseNode->appendPostOps(ops);
             continue;
         }
+        auto* fakeQuantizeNode = dynamic_cast<MKLDNNFakeQuantizeNode *>(node.get());
+        if (fakeQuantizeNode) {
+            fakeQuantizeNode->appendPostOps(ops);
+            continue;
+        }
```

Comment:
It is possible to avoid the expensive `dynamic_cast`. Just test `node->getType()` is equal to `MKLDNNPlugin::Type::FakeQuantize` than polymorphically call `appendPostOps` to the node pointer since this is a virtual method.

Reply:
I agree, but decided to leave the behavior similar to eltwise node above.